app/src/main/java/pl/imobitech/androidkurs/features/data/remote/model/CharacterRemote.kt

```diff
@@ -0,0 +1,37 @@
+package pl.imobitech.androidkurs.features.data.remote.model
+
+import com.google.gson.annotations.SerializedName
+import pl.imobitech.androidkurs.features.episodes.domain.model.Character
+
+
+data class CharacterRemote(
+    @SerializedName("id") val id: Int,
+    @SerializedName("name") val name: String,
+    @SerializedName("status") val status: String,
+    @SerializedName("species") val species: String,
+    @SerializedName("type") val type: String,
+    @SerializedName("gender") val gender: String,
+    @SerializedName("origin") val origin: OriginRemote,
+    @SerializedName("location") val location: LocationRemote,
```

Comment:
Tutaj obiekt `location` ma tylko dwa pola - `name` i `url`. Jeśli chcesz z tego korzystać w ten sposób, to konieczne będzie oznaczenie innych pól w klasie `LocationRemote` jako opcjonalne. Można też do tego podejść w inny sposób i stworzyć dedykowany model do tego obiektu, który będzie zawierał tylko te dwa pola.